examples/tabular/example_tabular_predictor_v2.py

```diff
@@ -0,0 +1,54 @@
+
+from autogluon.tabular import TabularPrediction as task
+from autogluon.tabular.task.tabular_prediction.predictor_v2 import TabularPredictorV2
```

Comment:
This will be shortened, the reason it is long is because we don't want it easily visible to users until it is ready.

We could potentially have the dataset loader be in predictor, but it is indeed a bit strange. We can discuss in future PRs.